### PR TITLE
Fix XSS in goToHome

### LINKED ISSUE #10: XSS in goToHome
1. https://www.code-labs.io/codelabs/about/index.html?index=javascript:alert(/xss/)
2. Click => Back arrow or Complete.

### PR DESCRIPTION
**Pull request by crhym3**
Closes #10 

R: @sirdarckcat, @ebidel

### CODE REVIEW

File: google-codelab.html [hunk 1]
```diff
@@ -437,12 +437,12 @@ <h1 class="title">{{title}}</h1>
           }
         }
 
-        var returnToIndex = index ? decodeURIComponent(index) : index;
-        if (!returnToIndex || returnToIndex.slice(-6) === '/index') {
-          returnToIndex = '/';
+        index = index ? decodeURIComponent(index) : '';
```
Comment (ebidel) [inline google-codelab.html]: Can you add a comment? It's starting to get unobvious at a glance.

**Comment by ebidel** [comment]
LGTM

File: google-codelab.html (hunk 1 above)
Comment (crhym3) [inline google-codelab.html]: how does it look now?

**Comment by ebidel** [comment]
I'll do a release

**Pull request merged**